views/404.handlebars

```diff
@@ -0,0 +1,5 @@
+<main role="main">
+    <div class="container center-content">
+        <h1>404 Not Found</h1>
+    </div>
+</main>
```

Comment:
add trailing newline.

Reply:
Will keep this in mind in the future lmao